packages/vaadin-lumo-styles/components.css

```
@@ -0,0 +1,3 @@
+@import './components/icon.css';
+@import './components/button.css';
+@import './components/input-container.css';
```

Nit: let’s keep it alphabetical.

```suggestion
@import './components/button.css';
@import './components/icon.css';
@import './components/input-container.css';
```